src/ui/components/pages/Catalog/CatalogLauncher/CatalogLauncher.tsx

```diff
@@ -328,6 +340,8 @@ export const CatalogLauncher = memo((props: Props) => {
     );
 });
 
+CatalogLauncher.displayName = CatalogLauncher.name;
```

Comment:
While it's good practice to give a displayName this won't work, the `CatalogLauncher` has no name (it's an arrow function).  

This is what you should do instead:  

`CatalogLauncher.displayName = symToStr({ CatalogLauncher });`  

and import `symToStr` like:  

```ts
import { symToStr } from "tsafe/symToStr";
```

Reply:
thx, I was looking for this function 😄